AdminWebsite/AdminWebsite/Services/UserAccountService.cs

```diff
@@ -269,17 +271,24 @@ public async Task DeleteParticipantAccountAsync(string username)
         }
 
         public async Task AssignParticipantToGroup(string username, string userRole)
-        {
-            await AddGroup(username, External);
-            
+        {   
             switch (userRole)
             {
                 case RepresentativeRole:
+                    await AddGroup(username, External);
                     await AddGroup(username, VirtualRoomProfessionalUser);
                     break;
                 case JohRole:
+                    await AddGroup(username, External);
```

Comment:
is it worth considering to use userRole=="Staff Member" ? await AddGroup(username, Internal) : await AddGroup(username, External)

Reply:
I believe that would work however the switch case reflects perfectly the treatment for the StaffMember, Representative and JohRole...the default will be always external